enumerate.rb

```diff
@@ -0,0 +1,21 @@
+module MyEnumerable
+  def all?
+    return false unless block_given?
+
+    each { |para| return false unless yield(para) }
+    true
+  end
+
+  def any?
+    return false unless block_given?
+
+    each { |para| return true if yield(para) }
+    false
+  end
+
+  def filter?
+    result = []
+    each { |para| result << para if yield(para) }
+    result
+  end
+end
```

Comment:
- [ ] according to the requirements there should be a list class, it seems you have overlooked the below 2 requirement points:
- Create a class MyList that has an instance variable @list.
- In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.